index.html

```diff
@@ -1,301 +1,217 @@
-  <form class="form" action="https://formspree.io/f/xpznodvr" method="post">
-  <ul class="form_input">
-    <li><input name="Fname" id="name" type="text" class="text"
-      maxlength="30"
-      placeholder="Enter Full Name"
-      required ></li>
-      <li>
-        <input name="Email" id="emial" type="email" class="text"
-      maxlength="30"
-      placeholder="Enter Email Address"
-      required >
-      </li>
-      <li>
-        <textarea
-        value="Wite your message"
-        id="messege"
-        maxlength="500"
-        placeholder="Please Enter Message"
-        required
-        cols="30"
-        rows="10"
-      ></textarea>
-      </li>
-      <li class="Get_btn">
-        <button type="summit" >Get In Touch</button>
-      </li>
-  </ul>
-  </form>
-  <div class="shape"><img src="./image/Shape.png" alt="Shape"></div>
-</div>
-</section>
-
-  </body>
-</html>
+            website look though some of my work and experiance! if you like what you
+            see and have a project you need coded, don't hestiate to conduct me.
+</p></div><div class="New_contact">
+<h2 class="connect">LET'S CONNECT</h2></div><ul class="social_media social_media_bottom ">
+<li class="social_media_list">
+    <a href="https://twitter.com/TamratYemsrach/">
+        <img src="./image/twitter.svg" alt="twitter"/></a>
+</li>
+<li class="social_media_list">
+    <a href="https://www.linkedin.com/in/yemsrach/"><img src="./image/Linkedin icon.svg" alt="Linkedin"/></a>
+</li></liclass="social_media_list"><li>
+<a href="/"><img src="./image/medium.svg" alt="medium"/></a></li><li class="social_media_list">
+<a href="https://github.com/Yemisirach"><img src="./image/github.svg" alt="github"/></a></li><li class="social_media_list"><img src="./image/Vector (1).png" alt="vector"></li></ul></div><button class="resume"><a href="#">Get my resume</a></button></div></div><div class="About_self_two"><div class="dropdawn"><div class="Language"><div class="vecor"><h2>Languages</h2><img width="10px" height="15px" class="open_vector" src="./image/Union.png" alt="vector"></div><div><ul class="language_list_all">
+<li class="language_list java"><img src="./image/Ellipse 1.png" alt="javaScript"><h4>JavaScript</h4>
+</li>
+<li class="language_list"><img src="./image/Ellipse 1 (1).png" alt="html"><h4>HTML</h4>
+</li>
+<li class="language_list"><img src="./image/Ellipse 1 (2).png" alt="css"><h4>CSS</h4>
+</li></ul></div></div></div><div class="framework Framework"><h2>Frameworks</h2><a href="#"><img src="./image/Union.png" alt="Union"></a></div><div class="framework"><h2>Skill</h2><a href="#"><img src="./image/Union.png" alt="Union"></a></div></div></section><!-- About section end --><section class="contact_me_section"><div class="Contact_me"><header class="contact_header"><h2>Contact me</h2></header><div class="contact_paragraph"><p>If you have an application you are insereted in developing, a
+              feature that you need built or a project that need coding. I'd
```

Comment:
- [ ] Instead of using `h2` here :blush: Please use `h1` because this is the main heading of your page and according to the [heading hierarchy](https://www.w3.org/WAI/test-evaluate/preliminary/#headings) :+1:
`<h1>Contact me</h1>`